gamefixes-gog/ulwgl-wheeloftime.py

```diff
@@ -0,0 +1,10 @@
+""" 
+The Wheel of Time
+"""
+#pylint: disable=C0103
+
+from protonfixes import util
+
+def main():
+    """ """
```

Comment:
Hi, do you mind removing this empty block comment? Same with the one in https://github.com/Open-Wine-Components/ULWGL-protonfixes/pull/36. 

It's useless and will be disallowed in the future once the linter gets merged. If you do that, I'll merge this and the other one ASAP

Reply:
Should be resolved in both PRs!